.github/workflows/validate.yml

```diff
@@ -111,11 +104,12 @@ jobs:
         with:
           fetch-depth: 0
       - uses: actions/setup-python@v4
-      - uses: akaihola/darker@1.5.1
+      - uses: akaihola/darker@0fb2501a3f6c1b2d64976afa57885aeec0601182
+        # pinned due to unreleased fix: https://github.com/akaihola/darker/issues/489
```

Comment:
The fix has been released, fwiw https://github.com/akaihola/darker/blob/master/CHANGES.rst#fixed-1